NitroxModel/Packets/VehicleMovement.cs

```diff
@@ -21,7 +19,7 @@ public VehicleMovement(String playerId, Vector3 playerPosition, Quaternion rotat
 
         public override string ToString()
         {
-            return "[VehicleMovement - playerId: " + PlayerId + " position: " + PlayerPosition + " Rotation: " + Rotation + " TechType: " + TechType + " guid: " + Guid + "]";
+            return "[VehicleMovement - TechType: " + TechType + " guid: " + Guid + "]\n\t" + base.ToString();
```

Comment:
Yes, the base class of `VehicleMovement` (which is `Movement`) already stringifies these fields, so it's better to reuse that functionality than to copy it.